x-pack/plugins/security_solution/cypress/tasks/rules_bulk_edit.ts

```diff
@@ -209,17 +222,15 @@ export const setScheduleLookbackTimeUnit = (timeUnit: TimeUnit) => {
 export const assertUpdateScheduleWarningExists = (expectedNumberOfNotMLRules: number) => {
   cy.get(RULES_BULK_EDIT_SCHEDULES_WARNING).should(
     'have.text',
-    `You're about to apply changes to ${expectedNumberOfNotMLRules} selected rules. The changes you made will be overwritten to the existing Rule schedules and additional look-back time (if any).`
+    `You're about to apply changes to ${expectedNumberOfNotMLRules} selected rules. The changes you make will overwrite the existing rule schedules and additional look-back time (if any).`
```

Comment:
Nit: would be nice to reduce the amount of text in this check by switching from `have.text` to `contain.text`.